packages/twenty-server/src/engine/core-modules/two-factor-method/two-factor-method.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Relation,
+  UpdateDateColumn,
+} from 'typeorm';
+
+import { UserWorkspace } from 'src/engine/core-modules/user-workspace/user-workspace.entity';
+
+@Entity({ name: 'twoFactorMethod', schema: 'core' })
+@ObjectType('TwoFactorMethod')
+export class TwoFactorMethod {
```

Comment:
I'm a bit confused, this table does provide any information? no type, token, etc. why is it useful then?

Reply:
It's because this PR is heavily WIP, nothing is really done. But:
- the token will be derived from the Primary Key of this table
- we only have one 2FA type which is TOTP, that's why we don't store it

Does it make sense?